src/windshaft/client.js

```diff
@@ -23,13 +23,12 @@ var WindshaftClient = function (options) {
   this.userName = options.userName;
   this.endpoint = options.endpoint;
   this.statTag = options.statTag;
-  this.forceCors = options.forceCors || false;
 
   this.url = this.urlTemplate.replace('{user}', this.userName);
 };
 
-WindshaftClient.DEFAULT_COMPRESSION_LEVEL = 3;
-WindshaftClient.MAX_GET_SIZE = 2033;
+WindshaftClient.prototype.MAX_URL_LENGTH = 2033;
```

Comment:
Aren't these just constants that only make sense within the definition scope? Is anyone using them?

I would declare the as `var MAX_URL_LENGTH = 2033;`, avoiding them to be available in the WindshaftClient instance.